core/src/main/scala/kafka/controller/Election.scala

```diff
@@ -17,13 +17,14 @@
 package kafka.controller
 
 import kafka.api.LeaderAndIsr
+import kafka.utils.Logging
 import org.apache.kafka.common.TopicPartition
 
 import scala.collection.Seq
 
 case class ElectionResult(topicPartition: TopicPartition, leaderAndIsr: Option[LeaderAndIsr], liveReplicas: Seq[Int])
 
-object Election {
+object Election extends Logging {
```

Comment:
I think this is correct but can you confirm that this is getting logged in the controller logs? See https://github.com/apache/kafka/blob/trunk/config/log4j.properties#L79-L80

Reply:
Hmm.  Good point. Will resubmit after testing.  Thanks.

Reply:
It does get logged in the controller log.

[2019-10-24 13:28:33,449] WARN Unclean leader election. Partition UleMe-393 has been assigned leader 0 from deposed leader -1. (kafka.controller.Election$)